src/shims/foreign_items.rs

```diff
@@ -1008,9 +1008,11 @@ trait EvalContextExtPriv<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
             "llvm.arm.hint" if this.tcx.sess.target.arch == "arm" => {
                 let [arg] = this.check_shim(abi, Abi::Unadjusted, link_name, args)?;
                 let arg = this.read_scalar(arg)?.to_i32()?;
+                // Note that different arguments might have different target feature requirements.
                 match arg {
                     // YIELD
                     1 => {
+                        this.expect_target_feature_for_intrinsic(link_name, Symbol::intern("v6"))?;
```

Comment:
Looks like `expect_target_feature_for_intrinsic` should take `&str` as its second argument?

Reply:
Yeah, since we only have `sse` and `avx` in `sym` anyways